guides/common/modules/con_upgrading-project-or-proxy-from-el8to9.adoc

```diff
@@ -0,0 +1,10 @@
+[id="upgrading_{project-context}_or_proxy_from_el8to9_{context}"]
+= Upgrading {Project} or {SmartProxy} from {EL} 8 to {EL} 9
```

Comment:
```suggestion
= Upgrading {EL} on {Project} or {SmartProxy}
```
This title would be more precise as to what the subject of the upgrade is.
Also shorter. :)
Remember to change the filename and ID as well.